ubuntu.sh

```diff
@@ -63,9 +65,15 @@ ensure_iface_naming_consistency() {
     then
       actual_if="$(ip -o -4 addr | grep -w ${VIP} | awk '{print $2}')"
       fix_iface_name "${actual_if}" "${EXTERNAL_IFACE}"
+      RENAME=true
     fi
   fi
 
+  if [ $RENAME = "true" ]
+  then
+    reboot
```

Comment:
We'll want to `exit` here, to prevent the script from continuing onward, and add an error code to prevent Odin from thinking the bootstrap is completed.